easybuild/easyblocks/g/gromacs.py

```diff
@@ -102,6 +103,12 @@ def get_gromacs_arch(self):
             # According to [2] the performance difference between SSE2 and SSE4.1 is minor on x86
             # and SSE4.1 is not supported by AMD Magny-Cours[1].
             res = 'SSE2'
+        elif optarch == OPTARCH_GENERIC and LooseVersion(self.version) >= LooseVersion('5.0'):
```

Comment:
@victorusu This version check is overkill, this is already handled where `get_gromacs_arch` is called?